lib/add_magic_comment.rb

```diff
@@ -16,11 +16,20 @@ module AddMagicComment
 
   def self.process(argv)
     directory = argv.first || Dir.pwd
+    ignore_pattern = Regexp.new(argv[1]) rescue nil
+    # Allow providing as either a Regex or a String
+    if ignore_pattern && ignore_pattern.is_a?(String)
```

Comment:
If `ignore_pattern` is truthy, it must be a `Regexp`, right?
Are there other tools that have a precedent for either taking a string or a regexp on the command line? If so, maybe we can follow their convention? Or if we can't think of any precedents, maybe use leading/trailing `/` to indicate a regexp?